labtool2.0/src/components/pages/ModifyCourseInstanceCodeReviews.js

```diff
@@ -179,6 +182,15 @@ export const ModifyCourseInstanceReview = props => {
     return Array.isArray(studentReviewWeeks) && !studentReviewWeeks.length
   }
 
+  const changeFilterTag = () => {
+    return async (e, data) => {
+      const { value } = data
+      const tag = props.tags.tags.find(tag => tag.id === value)
+      await props.filterByTag(tag)
+      props.filterStatesByTags({ tags: props.coursePageLogic.filterByTag, students: props.courseData.data })
+    }
+  }
```

Comment:
Same here, this could be just:
```
const changeFilterTag = async (e, data) => {
      const { value } = data
      const tag = props.tags.tags.find(tag => tag.id === value)
      await props.filterByTag(tag)
      props.filterStatesByTags({ tags: props.coursePageLogic.filterByTag, students: props.courseData.data })
    }
```